### DISCUSSION TITLE
Xarray as input

### DISCUSSION THREAD

**Discussion by MartinSJRogers** [TorchIO-project/torchio#1161] [Q&A]
Hi, is it possible to use an xarray as input to the torchio package. Alternatively, is there a function to convert an xarray into a torchio object for input?

I have searched for xarray in the discussions and issues, but couldn't find anything relevant- apologies if I have missed it. 

Many thanks in advance.

**Comment by fepegar** [comment]
Hi, @MartinSJRogers. This is not something we have considered. I'm not very familiar with xarray, but I see there's some metadata (e.g., `coords`) that could be leveraged automatically to create an affine with spacing information, which could be used by transform such as `Resample`.

Can you tell us a bit more about your use case?
